docs/api.md

```diff
@@ -91,9 +91,11 @@
     - [OrderSide](#xudrpc.OrderSide)
     - [Role](#xudrpc.Role)
   
+  
```

Comment:
Why?

Reply:
Auto generated by `npm run proto`. Not sure if it's worth manually removing.